x/claim/client/cli/query_claim_record.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/arkeonetwork/arkeo/x/claim/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
```

Comment:
what is this for?

Reply:
vestige of scaffolding, will remove.

Reply:
You can see the same in all of the arkeo module tx's as well 

https://github.com/arkeonetwork/arkeo/blob/b930a6dc6749599b1fccbe16a4b17a8e2a8b9d97/x/arkeo/client/cli/tx_close_contract.go#L15

I will remove all of them.

Reply:
[b617e11](https://github.com/arkeonetwork/arkeo/pull/29/commits/b617e11aeb8603c11f0eaeac1a0b286397ef05cd)